src/spec/CollisionShapeSpec.ts

```diff
@@ -116,6 +116,15 @@ describe('Collision Shape', () => {
       expect(pointTooFar).toBe(null, 'The circle should be too far away for a hit');
     });
 
+    it('can be raycast against only positive toi', () => {
```

Comment:
It might be more clear to avoid acronyms, or at least have their expansion next to them.